README.md

```diff
@@ -3,28 +3,9 @@
 
 Check out the [TCK Reference Guide](https://jakartaee.github.io/cdi-tck/) to get acquainted with the CDI TCK and learn how to execute and debug it.
 
-## Building CDI TCK artifacts
-To build the CDI TCK artifacts, use:
-
-`mvn -DtckAuditSourceGen install`
-
-or when compiling against staged Jakarta artifacts:
-
-`mvn -DtckAuditSourceGen -Pstaging install`
-
-## Building the CDI TCK distribution
-The CDI TCK distribution artifact is built by specifing an additional `-Drelease` property to build the TCK reference
-documentation and distribution bundle, e.g.:
-
-`mvn -DtckAuditSourceGen -Drelease install`
-
-## Eclipse Continuous Integration Environment
-The Eclipse continuous integration environment interface for the CDI project is located at https://ci.eclipse.org/cdi/
-The https://github.com/jakartaee/cdi/wiki/Eclipse-CI-Release-Jobs page describes the jobs found there.
-
 ## Sources in GIT
 
-Master branch contains the CDI TCK 4.0
+Master branch contains the work-in-progress on CDI TCK 4.0
```

Comment:
This isn't really true now, is it?

Reply:
@Ladicek good point, I really just did a revert of the original commit. I'll add these small bits back in.